airflow_clickhouse_plugin/sensors/clickhouse_sql_sensor.py

```diff
@@ -0,0 +1,61 @@
+from airflow.exceptions import AirflowException
+from airflow.sensors.sql_sensor import SqlSensor
+from airflow_clickhouse_plugin.hooks import clickhouse_hook
+
+
+class ClickHouseSqlSensor(SqlSensor):
+    def __init__(
+            self,
+            sql: str = None,
+            clickhouse_conn_id: str = 'clickhouse_default',
+            parameters=None,
+            database: str = None,
+            success=None,
+            failure=None,
+            fail_on_empty=False,
+            *args,
+            **kwargs,
+    ):
+        super(ClickHouseSqlSensor, self).__init__(
+            conn_id=clickhouse_conn_id,
+            sql=sql,
+            parameters=parameters,
+            success=success,
+            failure=failure,
+            fail_on_empty=fail_on_empty,
+            *args,
+            **kwargs,
+        )
+        self._database = database
+
+    def poke(self, context):
+        """
+        Monkey patch the original poke to use ClickHouseHook
```

Comment:
This is not a monkey patching, this is just method overriding. https://en.wikipedia.org/wiki/Monkey_patch

Just leave the link to initial code, this will be enough.